scheduler/strategy/strategy.go

```diff
@@ -16,13 +16,16 @@ type PlacementStrategy interface {
 }
 
 var (
-	strategies      map[string]PlacementStrategy
-	ErrNotSupported = errors.New("strategy not supported")
+	strategies              map[string]PlacementStrategy
+	ErrNotSupported         = errors.New("strategy not supported")
+	ErrNoResourcesAvailable = errors.New("no resources available to schedule container")
 )
 
 func init() {
 	strategies = map[string]PlacementStrategy{
-		"binpacking": &BinPackingPlacementStrategy{},
+		"binpacking": &BinpackPlacementStrategy{}, //compat
```

Comment:
doesn't have to happen in this PR but maybe a deprecation notice would be good. Considering that this strategy was the default I doubt anyone explicitly called out this strategy, but :+1: for keeping compat otherwise.